intezer_analyze_cli/utilities.py

```diff
@@ -7,28 +7,9 @@
 
 import click
 
-from intezer_analyze_cli.data_types import AnalysisErrorType
-
 log_file_path = ''
 
 
-def hash_sha256_file(path):
```

Comment:
I can't see where we used that so i guess that's why we deleted it?

Reply:
correct